x-pack/plugins/graph/public/services/outlink_encoders.js

```diff
@@ -6,38 +6,58 @@
 
 import rison from 'rison-node';
 
-export const outlinkEncoders = [{
+export const getOutlinkEncoders = i18n => [{
```

Comment:
I'm not sure, but maybe it is better just to import i18n from core in this file instead of passing it as param?

Reply:
I think we should use angular service in angular code